src/app/core/services/auth.service.ts

```diff
@@ -31,6 +33,7 @@ export class AuthService {
               private phaseService: PhaseService,
               private issueCommentService: IssueCommentService,
               private dataService: DataService,
+              private githubeventService: GithubEventService,
```

Comment:
There are quite a few instances where you used `githubeventService`. Can us camelCase instead? `githubEventService`